youtube_dl/extractor/mixcloud.py

```diff
@@ -85,9 +85,15 @@ def _real_extract(self, url):
         uploader_id = self._search_regex(
             r'\s+"profile": "([^"]+)",', webpage, 'uploader id', fatal=False)
         description = self._og_search_description(webpage)
-        like_count = str_to_int(self._search_regex(
-            r'\bbutton-favorite\b[^>]+m-ajax-toggle-count="([^"]+)"',
-            webpage, 'like count', fatal=False))
+
+        like_count = self._search_regex(
+            r'\bbutton-favorite[^>]+><[^>]+>([^<]+)',
+            webpage, 'like count', fatal=False)
+
+        if like_count:
+            like_count = like_count.strip()
+            like_count = 1000 * str_to_int(like_count[:-1]) if like_count.endswith('k') else str_to_int(like_count)
```

Comment:
This calculation is wrong. `str_to_int` just removes dots resulting in 11000 instead of 1100.